go-controller/pkg/node/node_ip_handler_linux.go

```diff
@@ -33,7 +33,7 @@ type addressManager struct {
 	// useNetlink indicates the addressManager should use machine
 	// information from netlink. Set to false for testcases.
 	useNetlink bool
-
+	syncPeriod time.Duration
```

Comment:
nit: add what you wrote on the commit message as a comment here, used for testing only or something

Reply:
Humbly, I don't think there's a need to explain this more. 
Its not used just for testing and its self explanatory what it accomplishes.

Reply:
ok I am fine with that, but note that the commit title said: `Node IP handler: allow sync period to be customizable for testing` which hinted that this was only added for tests.. My point was to at least add this is 30seconds by default or whatever as a comment for people to know info on sync.. but again its just a nit.